### PR TITLE
Updates Medical sprites, labcoats, biosuits, and several digitigrade sprites to match upstream

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Title
Speedmerging the medical resprite 2 mirror before I could do anything with it filled me with rage, which I used to fuel this PR

Medical and its unique items have ALL been brought to match the newly-added TG sprites:
- All have digitigrade variants
- Modular scrubs (red/white) match the new scrubs and can be toggled
- Bio-suit overrides removed in favor of TG's much cleaner and complete ones (Ours STILL didn't have fitting obj icons after MONTHS of use.)
- Labcoats removed in favor of TG's cleaner ones (less hard lines, cleaner sprites)
- Wholely re-sprites the Robotics labcoat so we keep our black/red colorscheme, but still have the better labcoat sprite base
- Digitigrade for the above 2, and muzzled variants for the bio-hoods
- Security Bio-Suit is, yes, red. However, the description states it as being an outdated version, as Armadyne nor Lopland work in biohazards, so it feels weird for them to have bio-suits. If players abuse the bio-suits to redsec, I'll replace them, but in the meantime I dont see it as urgent
- Updates the Digitigrade sprites for Radsuits and Firesuits to TG, because I was sick of digi people having the years-old ugly versions (Yes, the radsuit has a random emissive glitch, its TGs fault)

## How This Contributes To The Skyrat Roleplay Experience
image
image
image
dreamseeker_mxKwBhnTBK
image
(I tested it all but screenshotting them all would have rotted my brain as I got to testing at 5AM. Speaking of, dear god review this before merge in case I missed something.)

blessed be the clean sprites

## Changelog

:cl:
imageadd: Biosuits, labcoats, scrubs, and medical gear have all been updated to TG standard on humanoid AND digitigrade/muzzled crew! Robotics' labcoat has kept its black and red colors, dont worry. Firesuits and Radiation suits have been updated for digi/muzzled crew as well!
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 1]
```diff
@@ -63,27 +15,15 @@
...
-
-/obj/item/clothing/head/bio_hood/scientist
-	worn_icon = 'modular_skyrat/modules/aesthetics/clothing/head.dmi'
-
-/obj/item/clothing/head/bio_hood/janitor
-	worn_icon = 'modular_skyrat/modules/aesthetics/clothing/head.dmi'
-
-/obj/item/clothing/head/bio_hood/virology
-	worn_icon = 'modular_skyrat/modules/aesthetics/clothing/head.dmi'
+	desc = "A hood that protects the head and face from biological contaminants. This is a sligtly-outdated model from NanoTrasen Securities - you can hardly see through the foggy visor's ageing red. Hopefully it's still up to spec..."
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	desc = "A hood that protects the head and face from biological contaminants. This is a slightly outdated model from NanoTrasen Securities - you can hardly see through the foggy visor's ageing red. Hopefully it's still up to spec..."
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 2]
```diff
@@ -1,57 +1,9 @@
...
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
-
-/obj/item/clothing/suit/toggle/labcoat/virologist
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
-
-/obj/item/clothing/suit/toggle/labcoat/science
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	desc = "A suit that protects against biological contamination. This is a sligtly-outdated model from NanoTrasen Securities, using their red color-scheme and even outdated labelling. Hopefully it's still up to spec..."
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	desc = "A suit that protects against biological contamination. This is a slightly outdated model from NanoTrasen Securities, using their red color-scheme and even outdated labelling. Hopefully it's still up to spec..."
```

**Review by tf-4 — COMMENTED** [review]
God bless

**Comment by Gandalf2k15** [comment]
Actually the modular biosuits are still better.

**Comment by ghost** [comment]
> Actually the modular biosuits are still better.

Objectively false for a ton of reasons:
- Only had mob sprites
- Didn't have Digi sprites
- Flat, single-color, and an entirely different style of spriting - this applies to all, as they were all recolors of one
- Baked-in hood despite the Bio items having seperate hoods/suits. I hadnt ever even seen the Hood sprite work because of this.
- Did I mention, no Obj sprites, only Mob sprites, so it was confusing as hell to put on a green-accented biosuit and it looks yellow on a human. And again that the hood was baked in and jank as hell because of it.

I DO like the sprites, but the way they were ham-fisted in was just unacceptable. I have a plan with them, hence the sprites not being deleted, but the rest, as BIOsuits, should look like sci-fi-esque Biosuits, rather than modern day Bio/Chemical-suits.

**Comment by Gandalf2k15** [comment]
Then fix them up to be correct

**Comment by ghost** [comment]
I'm not fully re-spriting stolen sprites that are wholely out of place. It's out of scope for this PR as well.

**Review by GoldenAlpharex — APPROVED** [review]
Code and sprites check out for me!

**Comment by GoldenAlpharex** [comment]
I completely forgot to merge this.

**Comment by optimumtact** [comment]
Hey you missed the roboticist coat!

**Comment by ghost** [comment]
> Hey you missed the roboticist coat!

I actually didn't, plus I updated our modular ones to the new sprites but our chosen color scheme 😎 
Unless I messed up somewhere whoops

**Comment by optimumtact** [comment]
https://user-images.githubusercontent.com/2544323/191168029-cfccaa98-0cb1-424d-b9ca-8a73264969c0.mp4

**Pull request merged**